index.html

```diff
@@ -111,7 +111,7 @@ <h3>Multi-Post Stories<br>Gain+Glory</h3>
                     <li>JavaScript</li>
                     <li>html</li>
                 </ul>
-                <button onclick="location.href='#';">See Project</button>
+                <button onclick="revealDetails(1)">See Project</button>
```

Comment:
- [ ] Dear @MarsIncarnate, according to the main requirement, You need to remove static contents from your HTML and use the array of objects(projects) to render your projects dynamically. `The projects section is created dynamically with the information from the projects array`

##### we need this content to be rendered dynamically from JavaScript.
https://github.com/MarsIncarnate/Microverse_Portfolio/blob/74a324a2c550ea7acad396ae69b94d3c3ef239ca/index.html#L91-L104